src/pywink/test/devices/base_test.py

```diff
@@ -138,25 +141,23 @@ def test_all_devices_manufacturer_device_id_state_is_valid(self):
     def test_all_devices_device_manufacturer_is_valid(self):
         devices = get_devices_from_response_dict(self.response_dict, device_types.ALL_SUPPORTED_DEVICES)
         device_with_no_manufacturer = ["GoControl Thermostat", "New Shortcut", "Test robot", "August lock"]
+        skip_types = [WinkKey, WinkPowerStripOutlet, WinkCloudClockDial, WinkCloudClockAlarm]
```

Comment:
This should probably be a module-level constant.  Otherwise, the two instances of this list may fall out of sync going forward.